src/jsTest/kotlin/it/krzeminski/githubactionstyping/AjvTest.kt

```diff
@@ -0,0 +1,98 @@
+package it.krzeminski.githubactionstyping
+
+import ajv.Ajv
+import ajv.ValidateFunction
+import io.kotest.assertions.withClue
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.Matcher
+import io.kotest.matchers.MatcherResult
+import io.kotest.matchers.booleans.shouldBeTrue
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.should
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNot
+import io.kotest.mpp.env
+import node.buffer.BufferEncoding.Companion.utf8
+import node.fs.exists
+import node.fs.existsSync
+import node.fs.opendir
+import node.fs.readFile
+import node.fs.readdirSync
+import node.fs.stat
+import node.path.path
+import ajv.Options as AjvOptions
+
+private val schemaFile = env("schemaFile")!!
+private val catalogDir = env("catalogDir")!!
+private val externalDir = env("externalDir")!!
+private val goodDir = env("goodDir")!!
+private val badDir = env("badDir")!!
+
+private lateinit var validate: ValidateFunction
+
+class AjvTest : FunSpec({
```

Comment:
nit: let's give this test + file a better name, "ajv" isn't very informative. Like `AjvSchemaValidationTest`? If not, at least let's put a KDoc on this class explaining its point.